it/test/uk/gov/hmrc/pillar2submissionapi/UktrSubmissionISpec.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test.uk.gov.hmrc.pillar2submissionapi
+
+import org.scalatest.OptionValues.convertOptionToValuable
+import play.api.libs.json.Json
+import play.api.mvc.AnyContentAsJson
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import test.uk.gov.hmrc.pillar2submissionapi.UktrSubmissionISpec._
+import uk.gov.hmrc.http.HttpVerbs.POST
+import uk.gov.hmrc.pillar2submissionapi.base.IntegrationSpecBase
+import uk.gov.hmrc.pillar2submissionapi.controllers.routes
+
+class UktrSubmissionISpec extends IntegrationSpecBase {
+
+  "UKTR Submission" when {
```

Comment:
what happens with an empty body request?

Reply:
So do you mean a body is provided, but contains nothing? (ie. json would look like {} )?
Because I do have a test for _no request body_ but wondering if you're meaning a slightly different scenario:
![image](https://github.com/user-attachments/assets/e0df5a1f-c585-469f-b052-655dea2e4e6b)